src/scopes.rs

```diff
@@ -61,6 +61,12 @@ impl Scope {
     self.vars.get(id)
   }
 
+  pub fn var_by_ident(&self, ident: &ast_view::Ident) -> Option<&Var> {
+    self
+      .vars
+      .get(&(ident.inner.sym.clone(), ident.inner.span.ctxt()))
+  }
```

Comment:
This surely works, but how about the following? It looks more idiomatic IMO.

```suggestion
  pub fn var_by_ident(&self, ident: &ast_view::Ident) -> Option<&Var> {
    self.vars.get(&ident.inner.to_id())
  }
```

Reply:
thanks, but I don't understand why ctx.scope().var(&name.inner.to_id()) doesn't work :( in no_const_assign.ts file

Reply:
I fixed